vars/main.yml

```diff
@@ -1,4 +1,6 @@
 __timesync_chrony_version: "{{
-  ansible_facts.packages['chrony'][0].version|default('')|int }}"
+  (( ansible_facts.packages.chrony | default([]) )[0]
+    | default(dict()) )['version'] | default('0') }}"
 __timesync_ntp_version: "{{
-  ansible_facts.packages['ntp'][0].version|default('')|int }}"
+  (( ansible_facts.packages.ntp | default([]) )[0]
+    | default(dict()) )['version'] | default('0') }}"
```

Comment:
Is this a better way to fix this issue?  https://github.com/linux-system-roles/timesync/pull/67/commits/8772d38af1c03b5fa02549b7b73a5071c9fe2237